test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -1108,10 +1110,6 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthFo
           test.form.setPassword('pass');
           test.setNextResponse(resUnauthorized);
           test.form.submit();
-          var button = test.form.submitButton();
-          var buttonClass = button.attr('class');
-          expect(buttonClass).toContain('link-button-disabled');
-          expect(test.form.isDisabled()).toBe(true);
```

Comment:
In this case, it looks like the purpose of this test it to verify that we do enter the disabled state. This requires some work on our part - we need to provide a helper method to do something like:
1. Mock ajax request, but don't return the response
2. Have test validate expection, and then call a function to resolve the response promise

Can you comment out this test (with the original code where it is doing these assertions) and write a comment saying why this test no longer works? We'll fix it on our side after this PR gets checked in.